src/compat_wrappers.h

```diff
@@ -8,6 +8,10 @@
 
 #include "b-em.h"
 
+#ifndef WIN32
+#define HAVE_STPCPY
+#endif
```

Comment:
I was assuming that stpcpy was present on all non-windows platforms, and that we did't need to test for it in autotools. If that's a bad assumption, then you'll need to add that check, because I have no experience of autotools.